scw_serverless/cli.py

```diff
@@ -145,3 +146,31 @@ def deploy(
             sdk_client=client,
         )
         manager.update_routes()
+
+
+@cli.command()
+@CLICK_ARG_FILE
+@click.option(
+    "--port",
+    "-p",
+    "port",
+    default=8080,
+    show_default=True,
+    help="Set port to listen on.",
+)
+@click.option(
+    "--debug",
+    "-d",
+    "debug",
+    default=True,
+    show_default=True,
+    help="Run Flask in debug mode.",
+)
+def dev(file: Path, port: int, debug: bool) -> None:
+    """Run functions locally with Serverless Local Testing."""
+    app.Serverless = local_app.ServerlessLocal
+    scw_serverless.Serverless = local_app.ServerlessLocal
```

Comment:
This is of course very hacky but it's hard to find a better way as:
- the instantiation of the Serverless class is done in the client's code
- the binding to the handlers is done via decorators, and thus already done when we import a global instance so we need to "act" before that

My first approach was to add a handler parameter to the functions which would store the python handler as an object. This doesn't work well with multiprocessing (can't pickle a Python function) and I don't like it (may introduce side effects when deploying). Ideally, local testing should not modify the deploy command.

The second approach could have been a global variable `LOCAL_TESTING=False` that changes the behavior of the Serverless class. But this results in some conditionals imports as we do not want to import the local testing framework in app.py to avoid packaging issues when deploying.

So finally we override the class directly with a derived class. This is not ideal but it's quite clean compared to those alternatives.